bindata/assets/config/defaultconfig.yaml

```diff
@@ -19,6 +19,8 @@ extendedArguments:
   - "true"
   leader-elect-retry-period:
   - "3s"
+  leader-elect-renew-deadline:
+  - "12s" # This will increase timeout value from 5s to 6s that is required in case primary dns server in resolv.conf fail.    
```

Comment:
thanks for adding the message. I think it would rather make sense to either say `from default 5s` or remove `from 5s` entirely